harpy/_cli_types.py

```
@@ -0,0 +1,66 @@
+    name = "contig_list"
+    def convert(self, value, param, ctx):
+        # check if it's a file
+        if os.path.exists(value):
+            if not os.path.isfile(value):
+                self.fail(f"{value} is not a file.", param, ctx)
+            with open(value, "r") as cont_in:
+                return [i.strip() for i in cont_in.readlines()]
+        else:
+            return [i.strip() for i in value.split(',')]
+
+class ContigList(click.ParamType):
```

Ruff

Redefinition of unused `ContigList` from line 42

(F811)